doc/src/modules/physics/vector/advanced.rst

```diff
@@ -8,6 +8,48 @@ the features that will be implemented in the future will also be covered, along
 with unanswered questions about proper functionality. Also, common problems
 will be discussed, along with some solutions.
 
+Time Variable
+=============
+
+The objects in `sympy.physics.vector` and `sympy.physics.mechanics` rely on a
+default symbol for time. If symbols are created with ``dynamicsymbols()`` they
+are implicit functions of the default time variable. This default symbol can be
+accessed like so::
+
+   >>> from sympy.physics.vector import TIME
+
+The default behavior is::
+
+   >>> TIME
+   t
+   >>> from sympy.physics.vector import dynamicsymbols
+   >>> q = dynamicsymbols('q')
+   >>> q
+   q(t)
+   >>> list(q.free_symbols)[0] is TIME
```

Comment:
I don't think TIME should be treated as singletonized. It's just defined as `Symbol('t')`. Another `Symbol('t')` should also work. And in general, singletonization isn't necessary for SymPy objects, because they are immutable, so regular equality comparison works fine.